scripts/protoc-generator.sh

```diff
@@ -13,14 +13,18 @@ install_dependencies() {
 
     go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@v2.11.3
 
+    go install github.com/bufbuild/buf/cmd/buf@v1.8.0
+
     curl -sSo /tmp/protoc-gen-grpc-java https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/1.49.0/protoc-gen-grpc-java-1.49.0-linux-x86_64.exe
     chmod +x /tmp/protoc-gen-grpc-java
 }
 
 lint() {
-    local PROTO_DIR=${1:-.}
+    buf lint || exit 1
```

Comment:
This change makes it necessary to make the `lint` call come after `install_dependencies`. Otherwise scripts fail:
<img width="779" alt="image" src="https://user-images.githubusercontent.com/3210701/194009984-6ae3de23-c310-4f13-baf9-0e8b12a4aaf7.png">

Reply:
Good spot. I fixed it in a later PR but will pull that here.

Reply:
fixed